pkg/server/server.go

```diff
@@ -114,7 +126,12 @@ func (s *ImmuServer) Start() error {
 		return err
 	}
 
-	metricsServer := StartMetrics(s.Options.MetricsBind(), s.Logger)
+	metricsServer := StartMetrics(
+		s.Options.MetricsBind(),
+		s.Logger,
+		func() float64 { return float64(s.Store.CountAll()) },
+		func() float64 { return time.Since(startedAt).Hours() },
```

Comment:
is possible to use an inline time.Now() and avoid to use a var?

Reply:
@mmeloni the `startedAt` value needs to be set only once: when server was started. If a call to `time.Now()` would be used there, the duration returned by `time.Since` will always be almost zero (from `time.Since` godoc: It is shorthand for `time.Now().Sub(t)`, so it will basically compare now with now).